hack/choco/tools/chocolateyinstall.ps1

```diff
@@ -2,14 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 $ErrorActionPreference = 'Stop';
-$releaseVersion = 'v0.10.0'
+$releaseVersion = 'v0.11.0-rc.2'
```

Comment:
We gotta change this to `v0.11.0` 😅

Reply:
There is another PR doing this:
https://github.com/vmware-tanzu/community-edition/pull/3727
https://github.com/vmware-tanzu/community-edition/pull/3728